.appveyor/install.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+export PATH="/mingw64/bin:${PATH}"
+export PKG_CONFIG_PATH="/mingw64/lib/pkgconfig:${PKG_CONFIG_PATH}"
+export GOROOT=/mingw64/lib/go
+export GOPATH=/go
+export CGO_ENABLED=1
+
+pacman --noconfirm -S \
+    mingw64/mingw-w64-x86_64-go \
+    mingw64/mingw-w64-x86_64-libusb
+
+go version
+go get github.com/google/gousb/...
```

Comment:
you might need to use "-t" to download dependencies for the tests too.

Are you sure it doesn't overwrite the cloned version? If the package does not exist locally, then "go get" would grab the master branch, and we want appveyor to run tests for branches.